test/tables.dashdb.sql

```diff
@@ -1,40 +1,19 @@
-SET CURRENT SCHEMA = CURRENT USER;
```

Comment:
Yes. Except the create statements none of the statements are even required for the tests.